build_deploy.sh

```diff
@@ -31,8 +37,14 @@ else
     podman login -u="$RH_REGISTRY_USER" -p="$RH_REGISTRY_TOKEN" registry.redhat.io
     podman build -t "${IMAGE_NAME}:${IMAGE_TAG}" .
     podman push "${IMAGE_NAME}:${IMAGE_TAG}"
-    for TAG in "latest" "qa"; do
-        podman tag "${IMAGE_NAME}:${IMAGE_TAG}" "${IMAGE_NAME}:$TAG"
-        podman push "${IMAGE_NAME}:$TAG"
+
+    if [[ $GIT_BRANCH == *"security-compliance"* ]]; then
+        docker --config="$DOCKER_CONF" tag "${IMAGE}:${IMAGE_TAG}" "${IMAGE}:security-compliance"
+        docker --config="$DOCKER_CONF" push "${IMAGE}:security-compliance"
+    else
+        for TAG in "latest" "qa"; do
+            docker --config="$DOCKER_CONF" tag "${IMAGE_NAME}:${IMAGE_TAG}" "${IMAGE_NAME}:$TAG"
+            docker --config="$DOCKER_CONF" push "${IMAGE_NAME}:$TAG"
```

Comment:
Nice... just keep these four command using `podman` instead of `docker` so it works even without the wrapper installed.